### PR TITLE
Add ci test to run on changes

### PR DESCRIPTION
**Pull request by nh758**

### CODE REVIEW

File: lib/tasks/testSetup.js [hunk 1]
```diff
@@ -167,7 +167,11 @@ function waitClosed(done) {
    var response = shell.exec(`docker stack ls | grep ${Options.stack}`, {
       silent: true,
    });
-   if (response.stdout != "") {
+   //Need to wait for the network to be removed or waitBoot will fail.
+   var network = shell.exec(`docker network ls | grep ${Options.stack}`, {
```
Comment (Hiro-Nakamura) [inline lib/tasks/testSetup.js]: I noticed in the other PR that we used: ``` shell.exec().grep()``` for windows compatibility.  How about here?

**Review by Hiro-Nakamura — CHANGES_REQUESTED** [review]

File: lib/tasks/testSetup.js (hunk 1 above)
Comment (nh758) [reply lib/tasks/testSetup.js]: Sorry missed a commit

File: lib/install.js [hunk 2]
```diff
@@ -210,7 +210,7 @@ function checkProd2021(done) {
 function cloneRepo(done) {
    console.log("... cloning repo");
    shell.exec(
-      `git clone https://github.com/Hiro-Nakamura/ab_runtime.git ${Options.name}`
+      `git clone https://github.com/nh758/ab_runtime.git ${Options.name}`
```
Comment (Hiro-Nakamura) [inline lib/install.js]: I just moved this to ```digi-serve``` so we should probably change it to that.

File: lib/utils/dbInit.js [hunk 3]
```diff
@@ -21,7 +21,8 @@ function ProcessData(context, data) {
    data = data.toString();
 
    // this is helpful to know if mysql is up and running the 1st time
-   if (data.indexOf("mysqld: ready for connections.") > -1) {
+   if (data.indexOf("mysqld: ready for connections.") > -1 ||
+       data.indexOf("mariadbd: ready for connections") > -1) {
```
Comment (Hiro-Nakamura) [inline lib/utils/dbInit.js]: think we could just simplify these to a single  ```data.indexOf("ready for connections") > -1``` ?

**Review by Hiro-Nakamura — CHANGES_REQUESTED** [review]

**Pull request merged**